src/test/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCallerTest.scala

```diff
@@ -88,8 +88,8 @@ class VanillaUmiConsensusCallerTest extends UnitSpec with OptionValues {
     val caller = cc(cco(errorRatePreUmi=PhredScore.MaxValue, minReads=1, minConsensusBaseQuality=0.toByte))
     val consensus = caller.consensusCall(Seq(source))
     consensus shouldBe 'defined
-    consensus.get.bases shouldBe source.bases
-    consensus.get.quals shouldBe source.quals
+    consensus.value.bases shouldBe source.bases
```

Comment:
Why make these changes from `get` to `value`?  That's invoking a scalatest-specific implicit to allow you to call `.value` on `Option` since it's only defined on `Some` not `None`.